### ISSUE TITLE
Upload time and size don't line up with their headers

### ISSUE THREAD

**Issue by PixelOrange** [Salandora/OctoPrint-FileManager#39]
See screenshot for example. 
![octopifilemanager](https://user-images.githubusercontent.com/5042764/110723002-f3e93c80-81d8-11eb-9826-479f34d8ec1a.PNG)

Cause: The `<th>` is in its own table and the folder contents (including the back button) are in their own div and table. This causes the anchoring between columns to break.

Possible Fix: Move the back button above the headers.

**Comment by Salandora** [comment]
Hey,

Thanks for reporting the issue I just realized the same issue after releasing version 0.1.7rc1.

Problem here is the "Back" button it only consists of 1 <td> element and does not have the td's for uploaded and size which therefore throws the whole table out of alignment. 
Also the header and the conent being seperated is by design so that you can always see the header even when you need to scroll down.
I made a fixed table layout in CSS instead. 
This also opens the possibility to restyle the table by simply overriding my CSS. (Atleast that was one intention to do it this way)

Anyway long story short: Will be fixed in Version 0.1.7rc2.

**Comment by PixelOrange** [comment]
Closing
